### PR TITLE
feat: Update settings for production environment

### PR DESCRIPTION
**Pull request by jstlaurent**
## Changelogs

This PR updates the default settings for the client to use the new Clerk production environment.

For local dev, the client now supports using a `.env` file to point to Clerk's development environment. Use these values:
```.env
POLARIS_HUB_URL="http://localhost:3000/"
POLARIS_AUTHORIZE_URL="https://pure-whippet-77.clerk.accounts.dev/oauth/authorize"
POLARIS_TOKEN_FETCH_URL="https://pure-whippet-77.clerk.accounts.dev/oauth/token"
POLARIS_USER_INFO_URL="https://pure-whippet-77.clerk.accounts.dev/oauth/userinfo"
POLARIS_CLIENT_ID="QJg8zadGwjnr6nbN"
```

---

_Checklist:_

- [X] _Was this PR discussed in an issue? It is recommended to first discuss a new feature into a GitHub issue before opening a PR._
- [ ] ~_Add tests to cover the fixed bug(s) or the newly introduced feature(s) (if appropriate)._~
- [ ] ~_Update the API documentation if a new function is added, or an existing one is deleted._~
- [X] _Write concise and explanatory changelogs above._
- [X] _If possible, assign one of the following labels to the PR: `feature`, `fix` or `test` (or ask a maintainer to do it for you)._

---

_discussion related to that PR_

### CODE REVIEW

**Review by MichelML — APPROVED** [review]
lgtm

**Review by mercuryseries — APPROVED** [review]

**Comment by cwognum** [comment]
Looks good, but I wanted to check the `client_id`. How would I do this? Would that be through the [Backend API](https://clerk.com/docs/reference/backend-api/tag/OAuth-Applications) ?

File: polaris/hub/settings.py [hunk 1]
```diff
@@ -30,12 +30,12 @@ class PolarisHubSettings(BaseSettings):
 
     hub_url: HttpUrlString = "https://polarishub.io/"
     api_url: Optional[HttpUrlString] = None
-    authorize_url: HttpUrlString = "https://pure-whippet-77.clerk.accounts.dev/oauth/authorize"
+    authorize_url: HttpUrlString = "https://clerk.polarishub.io/oauth/authorize"
```
Comment (kiramclean) [inline polaris/hub/settings.py:33]: Does this increase the risk of devs accidentally doing things in prod or is there some other gate in place elsewhere that will prevent accidents?

**Comment by jstlaurent** [comment]
> Looks good, but I wanted to check the `client_id`. How would I do this? Would that be through the [Backend API](https://clerk.com/docs/reference/backend-api/tag/OAuth-Applications) ?

Yes, but you have to use an API key from the [production environment](https://dashboard.clerk.com/apps/app_2R7SvDFe0y858paHZ5Ulwc5mR4r/instances/ins_2gnqSWOdKhfAmjmUNDtbhMmHYZK/api-keys).

**Review by cwognum — APPROVED** [review]

File: polaris/hub/settings.py (hunk 1 above)
Comment (jstlaurent) [reply polaris/hub/settings.py:33]: That's a great point. I'll add support for using a `.env`file for local development, so it's easy to point to dev for us.

**Pull request merged**